webapp/packages/plugin-data-viewer/src/locales/zh.ts

```diff
@@ -46,6 +53,8 @@ export default [
   ['data_viewer_refresh_result_set', 'Refresh result set'],
   ['data_viewer_total_count_failed', 'Failed to get total count'],
   ['data_viewer_total_count_tooltip', 'Get total count'],
+  ['data_viewer_total_count_cancel_failed_title', 'Failed to cancel getting of total count'],
+  ['data_viewer_total_count_cancel_failed_desc', 'Counting already finished'],
```

Comment:
do we actually need to display this message or we can just display the conted number?